move-mutation-test/src/cli.rs

```diff
@@ -112,7 +111,7 @@ pub struct TestBuildConfig {
     ///
     /// Used mainly for disabling mutants with infinite loops.
     /// The default value is large enough for all normal tests in most projects.
-    #[clap(long = "gas-limit", default_value_t = 1_000_000)]
+    #[clap(long, default_value_t = 10_000)]
```

Comment:
This change could be in a separate commit with some justification as to why it's 100x smaller than it used to be 😄

Reply:
Done